app.test.js

```diff
@@ -387,4 +386,75 @@ describe("GET /api/users", () => {
             }
         })
     })
-})
\ No newline at end of file
+})
+
+describe("GET /api/users", () => {
+    test("returns an array of users with properties username, name and avatar url", () => {
+        return request(app)
+        .get("/api/users")
+        .expect(200)
+        .then(({body}) => {
+            for(const user of body.response){
+                expect(user).toEqual(
+                    expect.objectContaining({
+                        username: expect.any(String),
+                        name: expect.any(String),
+                        avatar_url: expect.any(String)
+                    })
+                )
+            }
+        })
+    })
+})
+
+describe("GET /api/articles?topic=", () => {
+    test("returns the articles with topic specified by the path query", () => {
+        return request(app)
+        .get("/api/articles?topic=mitch")
+        .expect(200)
+        .then(({body}) => {
+            const resBody = body.response
+
+            expect(typeof body).toBe("object")
+
+            resBody.forEach((item) => {
```

Comment:
Remember to include a length check if you're going to forEach through an array in a test.